pixeltable/env.py

```diff
@@ -816,6 +818,91 @@ def __init_spacy(self) -> None:
         except Exception as exc:
             raise excs.Error(f'Failed to load spaCy model: {spacy_model}') from exc
 
+    def _cleanup(self) -> None:
+        """
+        Internal cleanup method that properly closes all resources and resets state.
+        This is called before destroying the singleton instance.
+        """
+        # Close database connections and sessions
+        if self._current_session is not None:
+            try:
+                self._current_session.close()
+            except Exception as e:
+                _logger.error(f'Error closing session: {e}')
+            self._current_session = None
```

Comment:
What's the purpose in clearing the member variables like this? Won't they get garbage collected anyway once the singleton instance is discarded?

Reply:
we were using same Env object before(in Env.clear())
After moving to Env.init_env we don't need it as it creates new object will remove this now